.circleci/config.yml

```diff
@@ -165,14 +165,23 @@ jobs:
       - image: 'redislabsmodules/rmbuilder:latest'
     steps:
       - run:
-          name: Run QA Automation
+          name: Run QA Automation - RS version is master
           command: >-
             apt-get update &&
             apt-get -y install curl &&
             curl -k -u $QA_AUTOMATION_USERNAME:$QA_AUTOMATION_PASS -X POST
             -H "Content-Type: application/json"
             -d '{"service_id":"single_module_test_cycle_sanity_and_extended", "name":"redisgraph automation-testing", "properties":{"sut_version":"master", "email_recipients":"graph@redislabs.com", "sut_environments":[], "tools_environment":{}, "module_name": "RedisGraph", "module_version":"master", "cycle_environments_setup":[{"teardown":true, "name":"xenial-amd64-aws", "concurrency":1}, {"teardown":true, "name":"bionic-amd64-aws", "concurrency":1}]}}'
             https://qa-automation-center.redislabs.com/processes
+      - run:
+          name: Run QA Automation - RS version is 5.4.11
+          command: >-
+            apt-get update &&
+            apt-get -y install curl &&
```

Comment:
I don't think we need to perform these operation once again (not 100% sure)

Reply:
updated